xbmc/platform/darwin/tvos/input/LibInputTouch.mm

```diff
@@ -209,60 +209,41 @@ - (void)createPressGesturecognizers
 
 - (void)menuPressed:(UITapGestureRecognizer*)sender
 {
-  switch (sender.state)
```

Comment:
I'd prefer keep using `switch` (and adjust other places to use it) with single case and `default` - less lines to touch if another case needs to be added in the future.